sugar-iconify.md

```diff
@@ -0,0 +1,143 @@
+#Sugar Iconify
+
+##Description
+
+This script will create Sugar-compatible SVG icons from input.svg by
+adding the appropriate stroke and fill entities. It has two primary
+modes of operation: single-icon and icon set. For additional
+information about creating an SVG suitable for passing as the input
+to this script, see the tutorial on Making Sugar Icons.
+
+Single-icon: The default operation creates a single sugar icon — for
+instance an activity icon — from the input SVG. By default, the
+Sugarized icon will be written to a new file named input.sugar.svg,
+optionally in the output directory specified with -d. By invoking
```

Comment:
Maybe these option details should be after the list of options? Why highlight these particular options?